sdk/lusid/api_client.py

```diff
@@ -125,6 +138,9 @@ def __call_api(
             header_params = dict(self.parameters_to_tuples(header_params,
                                                            collection_formats))
 
+        header_params['X-LUSID-SDK-Language'] = 'Python'
+        header_params['X-LUSID-SDK-Version'] = '0.11.3028-MARK'          
```

Comment:
Has this moved from being on every API method to now only existing once in the API Client?

Reply:
I think so?

At least I couldn't find any where that you could set headers in `api.mustache` anymore, it's all in `api_client.mustache` now.

Reply:
I think this is the right approach, nice work!